### DISCUSSION TITLE
Bookmarks disapeared

### DISCUSSION THREAD

**Discussion by duncangroenewald** [iterate-ch/cyberduck#15457] [Q&A]
Just opened Cyberduck for Windows today and all my bookmarks have disappeared.  No history, nothing of any sort.

What the heck !

Version 8.7.1

**Comment by AliveDevil** [comment]
We changed some locations where these files are stored.
Please check in `%LocalAppData%\Packages\iterate.37637C3DE32E5_gr99yradmgg3r\LocalCache\Roaming\Cyberduck` and copy that directory to `%AppData%\Cyberduck`.

**Reply by dkocher** [reply]
To be resolved by #15514.
